Mathlib/Combinatorics/SimpleGraph/ConcreteColorings.lean

```diff
@@ -19,6 +19,16 @@ This file defines colorings for some common graphs
 
 namespace SimpleGraph
 
+theorem two_le_chromaticNumber_of_adj {α} {G : SimpleGraph α} {u v : α} (hAdj : G.Adj u v) :
+    2 ≤ G.chromaticNumber := by
+  refine le_of_not_lt ?_
+  intro h
+  have hc: G.Colorable 1 := chromaticNumber_le_iff_colorable.mp (Order.le_of_lt_add_one h)
+  let c : G.Coloring (Fin 1) := hc.some
+  have huv : c u ≠ c v := Coloring.valid c hAdj
+  rw [(c u).fin_one_eq_zero, (c v).fin_one_eq_zero] at huv
+  exact huv rfl
```

Comment:
```suggestion
  exact huv rfl
```
out of curiosity, I wonder if this works... Alternatively, `exact huv (Subsingleton.elim _ _)`, maybe?